src/Client.php

```diff
@@ -206,4 +238,104 @@ private function prepareEvent($payload, ?Scope $scope = null): ?Event
 
         return $event;
     }
+
+    /**
+     * Create an {@see Event} with a stacktrace attached to it.
+     *
+     * @param array<string, mixed>|Event $payload The data to be attached to the Event
+     */
+    private function buildEventWithStacktrace($payload): Event
+    {
+        if ($payload instanceof Event) {
+            return $this->buildEvent($payload);
```

Comment:
What about the stacktrace here?

Reply:
I will add a test case for this, this is not obvious how it works.

Reply:
Was dead code after all